app/extensions/torrent/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
What is this file for? There is no equivalent `app/extensions/brave/manifest.json`.